scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/SendMail.java

```diff
@@ -1,40 +1,78 @@
 package org.ow2.proactive.scheduler.util;
 
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
 import java.util.Properties;
 
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
 import javax.mail.Message;
 import javax.mail.MessagingException;
+import javax.mail.PasswordAuthentication;
 import javax.mail.Session;
 import javax.mail.Transport;
-import javax.mail.internet.AddressException;
-import javax.mail.internet.InternetAddress;
-import javax.mail.internet.MimeMessage;
+
 
 import org.apache.log4j.Logger;
 
+import org.ow2.proactive.scheduler.core.properties.PASchedulerProperties;
+
 
 public class SendMail {
 
-    private static Logger logger = Logger.getLogger(SendMail.class);
-    private static final String SMTP_HOST_PROPERTY_NAME = "mail.smtp.host";
-    private static final String DEFAULT_SMTP_HOST = "localhost";
-    private static final Session session;
+    public void sender(String to, String subject, String body) throws MessagingException {
+        final Properties properties = EmailConfiguration.getConfiguration().getProperties();
 
-    static {
-        Properties properties = System.getProperties();
-        if (properties.getProperty(SMTP_HOST_PROPERTY_NAME) == null) {
-            properties.setProperty(SMTP_HOST_PROPERTY_NAME, DEFAULT_SMTP_HOST);
-        }
-        session = Session.getDefaultInstance(properties);
-    }
+        Session session = Session.getInstance(properties, new javax.mail.Authenticator() {
+            protected PasswordAuthentication getPasswordAuthentication() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1161)